tests/model_selection/splitters/maxmin_split.py

```diff
@@ -0,0 +1,182 @@
+from typing import Union
+
+import pytest
+from rdkit import Chem
+from rdkit.Chem import Mol
+
+from skfp.model_selection.splitters.maxmin_split import (
+    maxmin_train_test_split,
+    maxmin_train_valid_test_split,
+)
+
+
+@pytest.fixture
+def all_molecules() -> list[str]:
+    return [
+        "OCC3OC(OCC2OC(OC(C#N)c1ccccc1)C(O)C(O)C2O)C(O)C(O)C3O",
+        "Cc1occc1C(=O)Nc2ccccc2",
+        "CCCCCCCCCCC",
+        "c1ccc2c(c1)ccc3c2ccc4c5ccccc5ccc43",
+        "CCCCCCCCCCCC",
+        "CCCCCCCCCCCCC",
+        "Clc1cc(Cl)c(c(Cl)c1)c2c(Cl)cccc2Cl",
+        "C1CCNCC1",
+        "ClC4=C(Cl)C5(Cl)C3C1CC(C2OC12)C3C4(Cl)C5(Cl)Cl",
+        "COc5cc4OCC3Oc2c1CC(Oc1ccc2C(=O)C3c4cc5OC)C(C)=C",
+    ]
+
+
+@pytest.fixture
+def additional_data() -> list[list[Union[str, int, bool]]]:
+    return [
+        ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j"],
+        [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+        [True, False, True, False, True, False, True, False, True, False],
+    ]
+
+
+def test_maxmin_split_size(all_molecules):
```

Comment:
Change name to `train_test_split_size`, following scaffold split test naming. We want to know from the test name which function we are testing and for what behavior